crates/symbolicator-service/src/services/download/sentry.rs

```diff
@@ -24,8 +24,54 @@ use crate::config::Config;
 use crate::utils::futures::CancelOnDrop;
 
 #[derive(Clone, Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
```

Comment:
Interesting, why do we use camel case here?

Reply:
This is for deserialization only, and this is what monolith returns. Eg. `"symbolType": "proguard"`

Reply:
Because: https://github.com/getsentry/sentry/blob/1023fa9522e2e12c96501475a95d65077079263c/src/sentry/api/serializers/models/debug_file.py#L15 :-)